WrightTools/data/_data.py

```diff
@@ -1565,6 +1577,109 @@ def remove_channel(self, channel, verbose=True):
         if verbose:
             print('channel {0} removed'.format(name))
 
+    def remove_variable(self, variable, verbose=True):
```

Comment:
Do we wish to have both singular and plural remove variable?
If so, this should just call the plural version with implied=False, I think, but leaning towards just the one function